legal-api/src/legal_api/models/amalgamation.py

```diff
@@ -19,11 +19,9 @@
 
 from enum import auto
 
-from sql_versioning import Versioned
-from sqlalchemy import or_
-from sqlalchemy_continuum import version_class
+from sql_versioning import Versioned, history_cls
```

Comment:
Updated `amalgamating_business` and `amalgamation` to use `sql_versioning` instead of `sqlalchemy_continuum`.